src/main/resources/static/scripts/dashboard.js

```diff
@@ -99,17 +98,51 @@
         } else if (!transactionType) {
             $("#typeTransactionError").text("Select type of transaction").show();
         }
+    }
+
+    function editTransactions() {
+        $.ajax("/transactions/" + $transactionId, {
+            type: "PUT",
+            data: {
+                id: $transactionId,
+                date: transactionDate,
+                name: $("#transactionName").val(),
+                amount: $("#amount").val(),
+                accountId: $(".highlight-account").attr("data"),
+                transactionType: transactionType
+            },
+            statusCode: {
+                200: function (response) {
```

Comment:
What will happen if the server responds with some other code? My concern is that the user could not understand what happens in case of an issue on the server-side.